### PR TITLE
Test: Create NFT page

### PR DESCRIPTION
**Pull request by prury**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Test

## Context

- [x] Related #7826 
- [ ] Requires deployment <snek/rubick/worker>

### <samp>🤖 Generated by Copilot at c79c41e</samp>

This pull request adds `data-testid` attributes to various components in the create nft feature to enable automated testing with Playwright. It also adds a new test file and a new method to the Commands class to test the creation of an nft and the opening of a new tab.

### <samp>🤖 Generated by Copilot at c79c41e</samp>

> _We are the testers of the nft world_
> _We use Playwright to unleash our wrath_
> _We add data-testid to every component_
> _We make them scream with our assert_

### CODE REVIEW

File: tests/e2e/createnft.spec.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { expect, test } from './fixtures'
+
+const prefix = ['ahk', 'ahp', 'ksm']
```
Comment (daiagi) [inline tests/e2e/createnft.spec.ts:3]: why  have an array if you only use 'ksm'

**Review by daiagi — APPROVED** [review]
looks good
maybe it is too much to ask, but it would be good to also test that the transaction is sent with the correct details
as if you were looking on the args in the wallet signing modal

**Comment by prury** [comment]
> looks good maybe it is too much to ask, but it would be good to also test that the transaction is sent with the correct details as if you were looking on the args in the wallet signing modal

yeah that would be great, but for this to be possible, we either have to merge #7849 or have a proper testnet, because i cannot proceed to the screen prior to the signing phase without balance on the account

File: tests/e2e/createnft.spec.ts (hunk 1 above)
Comment (prury) [reply tests/e2e/createnft.spec.ts]: removed

**Comment by yangwao** [comment]
> because i cannot proceed to the screen prior to the signing phase without balance on the account

wyt about setting some "dev" environment variable for codebase 👀

**Comment by prury** [comment]
> > because i cannot proceed to the screen prior to the signing phase without balance on the account
> 
> wyt about setting some "dev" environment variable for codebase 👀

using a dev seed? i think the limitations would be the same, no or low balance, no nfts/collections in it

**Review by preschian — APPROVED** [review]

**Pull request merged**